opensearch-operator/Dockerfile.oracle

```diff
@@ -0,0 +1,37 @@
+# Build the manager binary
+FROM  ghcr.io/oracle/oraclelinux:8-slim as builder
+
+RUN microdnf update -y \
+    && microdnf -y install golang \
+    && microdnf clean all
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+COPY pkg/   pkg/
+COPY opensearch-gateway/   opensearch-gateway/
+
+# Build
+ARG TARGETOS
+ARG TARGETARCH
+RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -a -o manager main.go
+
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM ghcr.io/oracle/oraclelinux:8-slim
```

Comment:
Probably should use https://github.com/orgs/verrazzano/packages/container/package/verrazzano-base instead of 8-slim